pom.xml

```diff
@@ -1541,9 +1542,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-        <pluginManagement>
-            <plugins>
+
```

Comment:
I guess I don't have the error because I am not using the Maven integration plugin. I'll try to ping some other people who are using Eclipse to see how they do.